charts/flagger/templates/deployment.yaml

```diff
@@ -102,6 +102,10 @@ spec:
               - --spider
               - http://localhost:8080/healthz
             timeoutSeconds: 5
+          {{- with .Values.env }}
+          env:
```

Comment:
Rather than doing this as an object, I think you'd be better served by using an array such that you enable people to use `configMapKeyRef` and `secretKeyRef`.  See [this example](https://github.com/helm/charts/blob/master/stable/external-dns/values.yaml#L270-L278) for more detail.

Reply:
@dcherman thanks a lot! using a `secretKeyRef` is the best way to deal with these tokens